lib/edit.js

```diff
@@ -2,7 +2,7 @@
 // open the package folder in the $EDITOR
 
 module.exports = edit
-edit.usage = 'npm edit <pkg>[@<version>]'
+edit.usage = 'npm edit <pkg>[/<subpkg>...]'
```

Comment:
I looked into how this summary got messed up, and it goes back to the edit docs having been wrong since 2011, and folks patching this usage summary to match them.

I would like to see `doc/cli/npm-edit.md` updated to match

Reply:
Will do, didn't see those were in the repo too.